src/components/SafeListSidebar/SafeList/index.tsx

```diff
@@ -69,23 +70,24 @@ const isNotLoadedViaUrl = ({ loadedViaUrl }: SafeRecordWithNames) => !loadedViaU
 export const SafeList = ({ onSafeClick }: Props): ReactElement => {
   const classes = useStyles()
   const networks = getNetworks()
-  const currentSafeAddress = extractSafeAddress()
-  const loadedSafes = useSelector(sortedSafeListSelector).filter(isNotLoadedViaUrl)
+  const currentSafeAddress = checksumAddress(extractSafeAddress())
+  const loadedSafes = useSelector(sortedSafeListSelector)
+    .map((loadedSafe) => ({ ...loadedSafe, address: checksumAddress(loadedSafe.address) }))
+    .filter(isNotLoadedViaUrl)
   const ownedSafes = useOwnerSafes()
   const localSafes = useLocalSafes()
 
   return (
     <StyledList>
       {networks.map(({ id, backgroundColor, textColor, label }) => {
         const isCurrentNetwork = id === getNetworkId()
-        const localSafesOnNetwork = localSafes[id]
         const ownedSafesOnNetwork = ownedSafes[id] || []
-        const shouldExpandOwnedSafes =
-          (isCurrentNetwork &&
-            ownedSafesOnNetwork.some(
-              (address) => address === currentSafeAddress && !isSafeAdded(loadedSafes, address),
-            )) ||
-          (!localSafesOnNetwork.length && ownedSafesOnNetwork.length <= MAX_EXPANDED_SAFES)
+        const localSafesOnNetwork = localSafes[id]
+          .map((localSafe) => ({ ...localSafe, address: checksumAddress(localSafe.address) }))
```

Comment:
This map should be on line 78.
But we shouldn't use `localSafes` for this, use `loadedSafes`.